catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/content/data/impl/ContentItemValidator.java

```diff
@@ -26,35 +24,52 @@
  * required by the {@link ddf.catalog.content.StorageProvider}.
  */
 public class ContentItemValidator {
-
     private static final Pattern QUALIFIER_PATTERN = Pattern.compile("\\w[a-zA-Z0-9_\\-]+");
```

Comment:
I can most certainly add a note about it here. Also, I do have explicit tests with comments that show we allow dash and dash-less id's which hopefully will help.